README.md

```diff
@@ -1,9 +1,9 @@
 redux
 =========================
 
-An experiment in fully hot-reloadable Flux.  
```

Comment:
Whitespace is significant in markdown files. Otherwise it'll remove the line breaks where I want them.